jwe.gemspec

```diff
@@ -21,4 +21,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rake'
   s.add_development_dependency 'simplecov'
   s.add_development_dependency 'codeclimate-test-reporter'
+  s.add_development_dependency 'base64'
```

Comment:
Isn't `base64` a runtime dependency?

Reply:
Ah yes, it's required as a runtime dependency here. Updated!

https://github.com/jwt/ruby-jwe/blob/03afe338df6fb37abdac9af4a0356bb9cbfb88de/lib/jwe.rb#L1